packages/pytest-simcore/src/pytest_simcore/helpers/postgres_tools.py

```diff
@@ -76,25 +76,46 @@ def migrated_pg_tables_context(
 def is_postgres_responsive(url) -> bool:
     """Check if something responds to ``url``"""
     try:
-        engine = sa.create_engine(url)
-        conn = engine.connect()
+        sync_engine = sa.create_engine(url)
+        conn = sync_engine.connect()
```

Comment:
Question: these sync functions. When we typically wait for postgres to be responsive, that guy runs in a container, and probably there are also other running as well. What is the point of doing it synchronously (potentially blocking the setup of other containers)? Same here, no need to change things now, but I believe that our tests are already pretty slow and annoying at that. Just wondering if we should not have everything async at this point.